### PR TITLE
Alma: Display correct information in call number.

### PR DESCRIPTION
**Pull request by EreMaijala**
Previously the ISBN was put into callnumber for getStatus(), which didn't seem right to me (in addition to being very book-specific). This PR switches it to information retrieved from the Alma API. For physical material the subfield d is call number. For others I substituted other useful information.

### CODE REVIEW

**Comment by demiankatz** [comment]
Thanks, @EreMaijala, I'll await approval from @betullam before merging.

**Review by betullam — APPROVED** [review]
@demiankatz @EreMaijala - Tested that with our production data. The suggestion from Ere seems correct for me too: It makes more sense to pull the data from the given subfields than adding an ISBN as callnumber.

**Comment by demiankatz** [comment]
Thanks, @betullam -- just merged!

**Pull request merged**
